euphonic/spectra.py

```diff
@@ -345,6 +346,27 @@ def from_dict(cls: S1D, d: Dict[str, Any]) -> S1D:
         return cls(d['x_data'], d['y_data'], x_tick_labels=d['x_tick_labels'],
                    metadata=d['metadata'])
 
+    @classmethod
+    def from_castep_phonon_dos(cls: S1D, filename: str,
+                               element: Optional[str] = None) -> S1D:
+        """
+        Reads DOS from a CASTEP .phonon_dos file
+
+        Parameters
+        ----------
+        filename
+            The path and name of the .phonon_dos file to read
+        element
+            Which element's PDOS to read. If not supplied reads
+            the total DOS.
+        """
+        data = read_phonon_dos_data(filename)
+        if element is None:
+            element = 'Total'
+        return Spectrum1D(data['dos_bins']*ureg(data['dos_bins_unit']),
+                          data['dos'][element]*ureg('dimensionless'),
```

Comment:
Are we aiming to sort out DOS dimensions in this PR or do it afterwards?

Reply:
I should be putting up an initial PR for the neutron weighted DOS soon, it would be good to discuss that there, for reasons which will become obvious

Reply:
Great!